torch/fx/examples/wrap_output_dynamically.py

```diff
@@ -0,0 +1,73 @@
+
+import torch
+from torch.fx import Proxy, GraphModule, symbolic_trace
+
+from enum import Enum, auto
+
+'''
+Wrap Graph output dynamically
+
+The following code demonstrates how change an existing Graph based on
+parameters specified at runtime. We'll let the user specify an
+activation function from a predefined Enum list, then we'll symbolically
+trace it. Next, we'll create a Proxy from the last operation in the
+Graph. We'll call our traced activation function with this Proxy and
+insert the ``output`` Node from that call into our Graph. (This final
+step will automatically inline the entire traced function.)
+'''
+
+
+# Sample module
+class M(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y):
+        y = torch.cat([x, y])
+        return y
+
+# Symbolically trace an instance of `M`
+traced = symbolic_trace(M())
+
+# Selected activation functions
+class ActivationFunction(Enum):
+    RELU = auto()
+    LEAKY_RELU = auto()
+    PRELU = auto()
+
+# Map activation function names to their implementation
+activation_functions = {
+    ActivationFunction.RELU: torch.nn.ReLU(),
+    ActivationFunction.LEAKY_RELU: torch.nn.LeakyReLU(),
+    ActivationFunction.PRELU: torch.nn.PReLU(),
+}
+
+def wrap_in_activation_function(m: GraphModule, fn: ActivationFunction) -> GraphModule:
+    # Get output node
+    output_node = next(iter(reversed(m.graph.nodes)))
+    assert output_node.op == "output"
```

Comment:
Instead of asserting that nodes[-1] is the output node, should probably iterated on `reversed(m.graph.nodes)` and assign to `output` node to an optional name, then assert that that's not none. Only a technicality, but `output` nodes don't necessarily need to be last (any code after an output node is not executed, but w/e)